ember-primitives/src/index.ts

```diff
@@ -4,6 +4,7 @@ if (macroCondition(isDevelopingApp())) {
   importSync('./components/violations.css');
 }
 
+export { Accordion, AccordionItem, AccordionHeader, AccordionContent, AccordionTrigger } from './components/accordion';
```

Comment:
because of the sub-components need quite a bit of wiring up, how do you feel about only exported `Accordion`?